src/Orleans.Connections.Security/Security/DuplexPipeStreamAdapter.cs

```diff
@@ -50,11 +49,6 @@ public override async ValueTask DisposeAsync()
             await Output.CompleteAsync();
         }
 
-        protected override void Dispose(bool disposing)
-        {
-            throw new NotSupportedException();
-        }
-#else
```

Comment:
is it by design that you deleted the Dispose in the NETCOREAPP branch?

Reply:
I felt that even though it should never be called, having a Dispose method which throws `NotSupportedException` is likely poor form